index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav>
+        <header>
+            <a href="#">
+                <img class="brang-img" src="./images/Brand-logo.png" alt="brang-logo" />
+            </a>
+            <img class="hamburger-menu" src="./images/Icon - Menu.png" alt="hamburger-menu">
+        </header>
+    </nav>
+    <div class="showcase">
```

Comment:
[ ] The container of the section is `a section tag`.It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. kindly fix that 🤓